Watchman.AwsResources.Tests/Services/Dax/DaxAlarmDataProviderTests.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Amazon.CloudWatch.Model;
+using Amazon.DAX.Model;
+using NUnit.Framework;
+using NUnit.Framework.Constraints;
+using Watchman.AwsResources.Services.Dax;
+
+namespace Watchman.AwsResources.Tests.Services.Dax
+{
+    [TestFixture]
+    public class DaxAlarmDataProviderTests

```

Comment:
Do you need all these tests? do they add much if you have the bigger overall tests cover the behaviour anyway? lots of tests on single classes makes refactoring harder. (I know there are already loads and appreciate you've followed existing patterns)